toolchest_client/tools/tool.py

```diff
@@ -297,7 +279,6 @@ def _handle_termination(self, signal_number, *_):
         if self.terminating:
             raise InterruptedError("Toolchest client force killed")
         self.terminating = True
-        sentry_sdk.set_tag('send_page', False)
```

Comment:
Is this removal intentional?

Reply:
Not as part of this PR. I'm guessing that this resulted from a merge from staging, but I'll investigate further

Reply:
Looks like this was from a merge commit from staging into this branch.